### PR TITLE
DX-3022 Optimize Sass

### PR DESCRIPTION
**Pull request by ckoegel**

### CODE REVIEW

File: site/src/css/components/_Carousel.scss [hunk 1]
```diff
@@ -4,23 +4,19 @@
...
     width: 100%;
-    height: 550px;
+    height: 36.2rem;
     display: flex;
     flex-direction: column;
     margin: 2em 0;
 
     @include mobile-view {
-        height: 300px;
+        height: 19.8rem;
```
Comment (bpateldx) [inline site/src/css/components/_Carousel.scss:13]: Thinking out loud here... can the style properties that are proportional (height, font-size) be declared as:
```
$base-font-size: 2.2em;
.base-container{
    font-size: $base-font-size;
    .something-smaller {
         font-size: $base-font-size - 0.2em; or $base-font-size - math.div($base-font-size , 0.11em)
    }
}
```
My syntax may not be accurate but hope it makes sense-:)

**Review by bpateldx — CHANGES_REQUESTED** [review]
Using Sass would be really beneficial to maintain different components with ease. This is awesome.

Did you end-up using any linter for sass files? If not, may be worth finding one. 

Please see if there is any opportunity to pull common properties out as commented.

File: site/src/css/components/_FeaturePanel.scss [hunk 2]
```diff
@@ -26,44 +26,44 @@
...
+    .panel-icon {
+        width: 3.3rem;
+        height: auto;
+    }
+    
+    .panel-title {
+        width: calc(100% - 3.3rem);
+        height: auto;
+        padding-left: 5px;
+        padding-top: 2px;
```
Comment (ajrice6713) [inline site/src/css/components/_FeaturePanel.scss:47]: 🤮

File: site/src/css/components/_SdkPage.scss [hunk 3]
```diff
@@ -29,75 +29,75 @@
...
+                }
+            }
 
-                    tr:nth-child(2n+1) {
-                        background-color: var(--cloud-accent);
+            tbody {
+                tr:last-child {
+                    /* rounded bottom left edge */
+                    td:first-child {
+                        border-radius: 0 0 0 20px;
```
Comment (ajrice6713) [inline site/src/css/components/_SdkPage.scss:84]:
```suggestion
                        border-radius: 0 0 0 🤮;
```

File: site/src/css/components/_SdkPage.scss [hunk 4]
```diff
@@ -29,75 +29,75 @@
...
                     }
+                    /* rounded bottom right edge */
+                    td:last-child {
+                        border-radius: 0 0 20px 0;
```
Comment (ajrice6713) [inline site/src/css/components/_SdkPage.scss:88]:
```suggestion
                        border-radius: 0 0 🤮 0;
```

File: site/src/css/components/modules/_CarouselSlide.scss [hunk 5]
```diff
@@ -7,76 +7,76 @@
         justify-content: center;
         align-items: flex-start;
 
-        .item {
+        .slide-item {
             background-color: var(--cloud-color);
             border-radius: 24px;
             box-shadow: 0px 0px 20px $black-shadow;
```
Comment (ajrice6713) [inline site/src/css/components/modules/_CarouselSlide.scss:13]: [cmon man](https://giphy.com/gifs/joe-biden-town-hall-SWoRKslHVtqEasqYCJ)

File: site/src/css/components/modules/_CarouselSlide.scss [hunk 6]
```diff
@@ -7,76 +7,76 @@
...
-                margin-top: 20px;
-                background-size: cover;
-                border-radius: 14px;
-                box-shadow: 0px 0px 10px $black-shadow;
-            }
+        .slide-image {
+            margin-top: 20px;
+            background-size: cover;
+            border-radius: 14px;
+            box-shadow: 0px 0px 10px $black-shadow;
```
Comment (ajrice6713) [inline site/src/css/components/modules/_CarouselSlide.scss:24]: output-onlineimagetools

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Review by matthewkmartin — APPROVED** [review]

**Pull request merged**